Coop_Listing_Site/Coop_Listing_Site/Views/ControlPanel/AddDepartment.cshtml

```diff
@@ -42,3 +60,78 @@
 <div>
     @Html.ActionLink("Back to List", "Index")@*can't seem to make this a button*@
 </div>
```

Comment:
> `@Html.ActionLink("Back to List", "Index")@*can't seem to make this a button*@` 

Would this work??
`<button value='@Html.ActionLink("Back to List", "Index")'></button>`